docker-compose.yml

```diff
@@ -39,6 +37,7 @@ services:
         - "8080:8080"
       depends_on:
         - node
+        - php-fpm
```

Comment:
I may have overwritten somewhere mate, this will cause an error, a loop, due to this line
https://github.com/ministryofjustice/justice-gov-uk/blob/main/docker-compose.yml#L21C9-L22C35

We also need port 80 listed in the local/server.conf for the integration above to work. 

This is all relevant unless something has changed 😄

Reply:
... and this PR is probably that lol

Reply:
haha! yup!